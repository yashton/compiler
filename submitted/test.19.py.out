(PUNCT "(")
(KEYWORD if)
(ID "string")
(PUNCT "==")
(ERROR "Illegal character '"can I do\
this?"' on line 1)
