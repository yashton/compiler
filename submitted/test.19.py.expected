(PUNCT "(")
(KEYWORD if)
(ID "string")
(PUNCT "==")
(LIT "can I dothis?")
(ENDMARKER)
